pennylane/ops/op_math/__init__.py

```diff
@@ -14,10 +14,38 @@
 """
 This module contains classes and functions for Operator arithmetic.
 
+Constructor Functions
+~~~~~~~~~~~~~~~~~~~~~
+
+.. currentmodule:: pennylane.ops.op_math
+
+.. autosummary::
+    :toctree: api
+
+    ~adjoint
+    ~ctrl
+    ~op_sum
+    ~prod
+    ~s_prod
```

Comment:
I assume all of these are top level imports (e.g., `qml.adjoint()`, `qml.ctrl()`, etc.)? If so, care needs to be taken here:

- The generated autosummary table should link to the top level `qml.adjoint()` docstrings
- We don't want Sphinx to generate _multiple_ docstrings for the same function, e.g., `qml.adjoint()` and `qml.ops.op_math.adjoint()`

Reply:
Yep. These should all be top-level imports. That was why I was getting errors about `s_prod` earlier in this PR.

Reply:
Got it 👍 Yeah noticed the latter happening:

https://pennylane--2730.org.readthedocs.build/en/2730/code/api/pennylane.prod.html

https://pennylane--2730.org.readthedocs.build/en/2730/code/api/pennylane.ops.op_math.prod.html

Reply:
Ahh nice catch!

Reply:
I replaced the links for to auto summary to specifically point to `~.pennylane.ops.op_math.prod`

Reply:
Swapped back to top level, and routed the docstring pages in `ops/op_math/` to top level as well 👍🏼